src/openforms/conf/base.py

```diff
@@ -699,6 +699,14 @@
         "task": "openforms.forms.admin.tasks.clear_forms_export",
         "schedule": crontab(hour=0, minute=0, day_of_week="sunday"),
     },
+    "activate-forms": {
+        "task": "openforms.forms.admin.tasks.activate_forms",
+        "schedule": crontab(minute="*"),
```

Comment:
I wonder if it is necessary to make it check every minute? :thinking: Would every hour not be sufficient? Although then we would have to make corresponding changes in the frontend to make it clear that it is only checked every hour :sweat_smile:

Reply:
IMO since the user can define the minutes as well, it should be every minute. @sergei-maertens can share his thoughts with us.

Reply:
Let's just do every minute and see if that load causes issues. If it does, I think I would rather tap into the `post_save` of the form and do a `activate_form.apply_async(eta=form.activate_on)` or run the beat command every hour and have that schedule the actual activation in x minutes.

For now I thought about not doing this because the task could be scheduled to run in 3 months, and that task/key will definitely expire in redis or has too much risk to be lost anyway.